testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/singleton/service/ValueServiceActivator.java

```diff
@@ -48,7 +50,32 @@ public class ValueServiceActivator implements ServiceActivator {
     public void activate(ServiceActivatorContext context) {
         ServiceBuilder<?> builder = context.getServiceTarget().addService(ServiceName.JBOSS.append("test", "service", "installer"));
         Supplier<SingletonPolicy> policy = builder.requires(ServiceName.parse(SingletonDefaultRequirement.SINGLETON_POLICY.getName()));
-        Consumer<ServiceTarget> installer = target -> policy.get().createSingletonServiceBuilder(SERVICE_NAME, new ValueService<>(new ImmediateValue<>(Boolean.TRUE)), new ValueService<>(new ImmediateValue<>(Boolean.FALSE))).build(context.getServiceTarget()).install();
+        Consumer<ServiceTarget> installer = target -> policy.get().createSingletonServiceBuilder(SERVICE_NAME, new ValueService<Boolean>(() -> Boolean.TRUE), new ValueService<Boolean>(() -> Boolean.FALSE)).build(context.getServiceTarget()).install();
```

Comment:
Similar comments to the above, but this is just a test class, so I wouldn't worry about it.

Reply:
Also fixed